crates/wdk-sys/build.rs

```diff
@@ -322,6 +368,42 @@ fn generate_hid(out_path: &Path, config: &Config) -> Result<(), ConfigError> {
     }
 }
 
+fn generate_parallel_ports(out_path: &Path, config: &Config) -> Result<(), ConfigError> {
+    cfg_if::cfg_if! {
+        if #[cfg(feature = "parallel-ports")] {
+            info!("Generating bindings to WDK: parallel_ports.rs");
+
+            let header_contents = config.bindgen_header_contents([ApiSubset::Base, ApiSubset::Wdf, ApiSubset::ParallelPorts]);
+            trace!(header_contents = ?header_contents);
+
+            let bindgen_builder = {
+                 let mut builder = bindgen::Builder::wdk_default(config)?
+                .with_codegen_config((CodegenConfig::TYPES | CodegenConfig::VARS).complement())
+                .header_contents("parallel-ports-input.h", &header_contents);
+
+                // Only allowlist files in the parallel-ports-specific files to avoid duplicate definitions
+                for header_file in config.headers(ApiSubset::ParallelPorts)
+                {
```

Comment:
nit: follow brace style to be consistent with the rest of the file

Reply:
ah thanks for the callout. rustfmt (and the pipeline task) missed this because the code is within the cfg_if macro. i've cleaned up this and other formatting mishaps!